students/jongmin/users/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path 
+from users.views import Signup
```

Comment:
```suggestion
from django.urls import path 

from users.views import Signup
```
- 파이썬 자체 모듈
- Django 모듈
- 내가 만든 클래스
위와 같은 순서대로 import 하는 객체들을 정렬해주시고, 파이썬과 장고, 장고와 내가 만든 클래스들 사이에 한 줄씩 띄워주세요.